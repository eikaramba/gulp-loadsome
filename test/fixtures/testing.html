<script src="http://www.loadso.me/jquery@2.1.1/angular@1.3.0/min.js"></script>
<script src="https://loadso.me/jquery@2.1.1/angular@1.3.0/min.js"></script>
<script src="//loadso.me/jquery@2.1.1/angular@1.3.0/min.js"></script>
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>