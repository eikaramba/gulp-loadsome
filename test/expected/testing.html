<script src=".tmp/2104bdb37e8faf5ad9e4ffbb0e045f6b.js"></script>
<script src=".tmp/2104bdb37e8faf5ad9e4ffbb0e045f6b.js"></script>
<script src=".tmp/2104bdb37e8faf5ad9e4ffbb0e045f6b.js"></script>
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>